samcli/lib/providers/sam_base_provider.py

```diff
@@ -17,6 +17,90 @@ class SamBaseProvider:
     Base class for SAM Template providers
     """
 
+    SERVERLESS_FUNCTION = "AWS::Serverless::Function"
```

Comment:
This is just refactoring and moving stuff around, am I right?

Reply:
Yup.